code/modules/clothing/masks/gasmask.dm

```diff
@@ -209,6 +209,14 @@
 	can_flip = 0
 	canstage = 0
 
+obj/item/clothing/mask/gas/lola
```

Comment:
This is not standard but all other items in the file start with `/`
My autism is midly annoyed by this